### DISCUSSION TITLE
Virtual Machine won't start

### DISCUSSION THREAD

**Discussion by Pandagineer** [bitcraze/discussions#242] [Q&A]
Hello. I had the Bitcraze VM installed (using Oracle Virtualbox), and everything was going fine for several weeks. Just a moment ago I tried installing ROSPY, and in the middle of the install my VM went black. I quit the VM, and restarted my computer, but I cannot restart the VM. Please help. (If I reinstall Virtualbox, will my VM configuration still be saved?)

**Comment by krichardsson** [comment]
It is hard to give any support for this, maybe there is a support forum for Virtualbox that can help?

**Reply by Pandagineer** [reply]
I understand. Just thought you had run into this before. I'll reach out to Virtualbox. Thanks.
